test/source/TestAlgorithm.cpp

```diff
@@ -2339,6 +2341,48 @@ int TestAlgorithm()
 		}
 	}
 
+	// issue #92
+	// disable in MSVC2013 because eastl::is_copy_constructible is always true...
+	#ifndef EA_COMPILER_MSVC_2013
+		{
+			eastl::vector<eastl::unique_ptr<int>> vec;
+			eastl::sort(vec.begin(), vec.end(),  [](const eastl::unique_ptr<int>& lhs, const eastl::unique_ptr<int>& rhs) { return *lhs < *rhs; });
+		}
+		{
+			eastl::vector<eastl::unique_ptr<int>> vec;
+			vec.emplace_back(new int(7));
+			vec.emplace_back(new int(-42));
+			vec.emplace_back(new int(5));
+			eastl::sort(vec.begin(), vec.end(),  [](const eastl::unique_ptr<int>& lhs, const eastl::unique_ptr<int>& rhs) { return *lhs < *rhs; });
+			EATEST_VERIFY(*vec[0] == -42);
+			EATEST_VERIFY(*vec[1] == 5);
+			EATEST_VERIFY(*vec[2] == 7);
+		}
+		{
+			for(unsigned tests=0; tests<500; ++tests)
```

Comment:
I was imitating the "strategy" implemented by random sorting test at line 2177. The principal appeared to be:
 * sort a small array
 * do the sorts many times to gain more confidence.